packages/network/src/v2/schemas/tableMetadata.ts

```diff
@@ -68,6 +68,14 @@ export function registerMetadata(
     }
     const decoded = decodeData(metadataSchema, metadataRecord);
     const tableName = decoded[0];
+    // TODO: decide if we still need table name in metadata since its part of the table ID now
```

Comment:
table id = table name is a `World` convention, table metadata exists on the `Store` level. Someone could use `Store` without `World` with table ids that follow a different convention, but still take advantage of all our `Store` tools and infra (eg MODE)